playbooks/robusta_playbooks/krr.py

```diff
@@ -37,6 +38,7 @@
 KRR_MEMORY_LIMIT: str = os.getenv("KRR_MEMORY_LIMIT", "2Gi")
```

Comment:
Putting here the comment just to remember to update the version before we merge it.